hugetlb.go

```diff
@@ -13,7 +13,7 @@ import (
 func NewHugetlb(root string) (*hugetlbController, error) {
 	sizes, err := hugePageSizes()
 	if err != nil {
-		return nil, nil
+		return nil, err
 	}
```

Comment:
this is not a critical error in runc too, we just let sizes be an empty slice and continue.

We cant get any stats in that case though but the system is probably not in the correct state either.